PublishedModules/ProjectAlice/AliceCore/AliceCore.py

```diff
@@ -473,15 +466,14 @@ def confirmModuleReboot(self, session: DialogSession, **_kwargs):
 
 
 	def reboot(self, session: DialogSession, **_kwargs):
-		value = 'greet'
-		if commons.isYes(session):
-			value = 'greetAndRebootModules'
+		value = 'greetAndRebootModules' if commons.isYes(session) else 'greet'
 
 		self.ConfigManager.updateAliceConfiguration('onReboot', value)
 		self.endDialog(session.sessionId, self.randomTalk('confirmRebooting'))
 		self.ThreadManager.doLater(interval=5, func=subprocess.run, args=[['sudo', 'shutdown', '-r', 'now']])
 
 
+	#TODO return value should be a dict according to Module.py, so it appears something is wrong here
```

Comment:
I did not change this, but I think it is not correct with the new dialog mapping anymore

Reply:
Looks wrong yes